PlanMyTrip/PlanMyTrip/SelectServicesViewController.m

```diff
@@ -1,13 +1,22 @@
 //
 //  SelectServicesViewController.m
-//  
+//  PlanMyTrip
 //
 //  Created by Farida Abdelmoneum on 7/6/22.
 //
 
 #import "SelectServicesViewController.h"
 
 @interface SelectServicesViewController ()
+@property (nonatomic) BOOL checkedH;
+@property (nonatomic) BOOL checkedF;
+@property (nonatomic) BOOL checkedC;
+@property (weak, nonatomic) IBOutlet UIButton *carsB;
+@property (weak, nonatomic) IBOutlet UIButton *flightsB;
+@property (strong, nonatomic) IBOutlet UIButton *hotelsB;
+- (IBAction)carsCheck:(id)sender;
+- (IBAction)flightsCheck:(id)sender;
+- (IBAction)hotelsCheck:(id)sender;
```

Comment:
there is a specific way we name the action handler and it usually looks like "[subject][Action]". In your case, `carsButtonTapped:(id)sender;`